includes/group-sites.php

```diff
@@ -1578,6 +1578,37 @@ function( $v ) use ( $source_site_url, $source_site_upload_dir, $dest_site_url,
 					}
 				}
 
+				// Updated custom nav menu items
+				$locations = get_theme_mod( 'nav_menu_locations' );
+				$menu_id   = isset( $locations['primary'] ) ? (int) $locations['primary'] : 0;
+				$nav_items = get_term_meta( $menu_id, 'cboxol_custom_menus', true );
+
+				if ( $menu_id && ! empty( $nav_items ) ) {
+					$group_type = cboxol_get_group_group_type( $group_id );
+
+					// Update Group Profile URL.
+					wp_update_nav_menu_item(
+						$menu_id,
+						$nav_items['group'],
+						array(
+							'menu-item-title'    => '[ ' . $group_type->get_label( 'group_home' ) . ' ]',
+							'menu-item-url'      => bp_get_group_permalink( $group ),
+							'menu-item-position' => 1,
+						)
+					);
+
+					// Update home URL.
+					wp_update_nav_menu_item(
+						$menu_id,
+						$nav_items['home'],
+						array(
+							'menu-item-title'    => __( 'Home', 'cbox-openlab-core' ),
+							'menu-item-url'      => home_url( '/' ),
+							'menu-item-position' => 1,
+						)
+					);
+				}
```

Comment:
I like how this is handled in OpenLab code more, unfortunately it's not doable here.

Side note: If we'll have time in the future, I would like to consolidate both Copying/Cloning into a single class.